_migrations/getting-started-data-migration.md

```diff
@@ -0,0 +1,331 @@
+        "migrationAssistanceEnabled": true
+    }
+    }
+    ```
+
+    The source and target cluster authorization can be configured to have no authorization, `basic` with a username and password, or `sigv4`. 
+
+3. Bootstrap the account with the following command:
+
+    ```bash
+    cdk bootstrap --c contextId=migration-assistant --require-approval never 
+    ```
+
+4. Deploy the stacks:
+
+    ```bash
+    cdk deploy "*" --c contextId=migration-assistant --require-approval never --concurrency 5
+    ```
+
+5. Verify that all CloudFormation stacks were installed successfully.
+
+### RFS parameters
+
+If you're creating a snapshot using migration tooling, these parameters are auto-configured. If you're using an existing snapshot, modify the  `reindexFromSnapshotExtraArgs` setting with the following values:
+
+    ```bash
+    --s3-repo-uri s3://<bucket-name>/<repo> --s3-region <region> --snapshot-name <name>
+    ```
+
+You will also need to give access to the `migrationconsole` and `reindexFromSnapshot` taskRole permissions to the EC2 bucket. 
+
+---
+
+## Step 5: Deploying the migration assistant
+
+To deploy the migration assistant, use the following steps:
+
+1. Bootstrap the account:
+   
+    ```bash
+    cdk bootstrap --c contextId=migration-assistant --require-approval never --concurrency 5
+    ```
+2. Deploy the stacks when `cdk.context.json` is fully configured:
+   
+    ```bash
+    cdk deploy "*" --c contextId=migration-assistant --require-approval never --concurrency 3
+    ```
+
+These commands deploy the following stacks:
+
+* Migration assistant network stack
+* Reindex From Snapshot stack
+* Migration console stack
+
+---
+
+## Step 6: Accessing the migration console
+
+Run the following command to access the migration console:
+
+```bash
+./accessContainer.sh migration-console dev <region>
+```
+
+
+`accessContainer.sh` is located in `/opensearch-migrations/deployment/cdk/opensearch-service-migration/` on the bootstrap instance. To learn more, see [Accessing the migration console]:
+`{: .note}
+
+---
+
+## Step 7: Checking the connection to the source and target clusters
+
+To verify the connection to the clusters, run:
+
+```bash
+console clusters connection-check
+```
+
+You should receive the following output:
+
+```
+* **Source Cluster:** Successfully connected!
+* **Target Cluster:** Successfully connected!
+```
+
+To learn more about migration console commands, see [Migration commands].
+
+---
+
+## Step 8: Snapshot creation
+
+Run the following to initiate creating a snapshot from the source cluster:
+
+```bash
+console snapshot create [...]
+```
+
+To check on the progress of the snapshot creation, use:
+
+```bash
+console snapshot status [...]
+```
+
+To learn more details about the snapshot, use:
+
+```bash
+console snapshot status --deep-check [...]
+```
+
+Wait for the snapshot to complete before moving to step 9.
+
+To learn more about snapshot creation, see [Snapshot Creation]:
```

Comment:
```suggestion
To learn more about snapshot creation, see [Snapshot Creation].
```

Reply:
Will add link after addressing the "Phases of migration" sections in the next PR.